test/ng/compileSpec.js

```diff
@@ -4219,6 +4227,299 @@ describe('$compile', function() {
     });
 
 
+    describe('one-way binding', function() {
+      it('should update isolate when the identity of origin changes', inject(function() {
+        compile('<div><span my-component ow-ref="obj">');
+        expect(componentScope.owRef).toBe(undefined);
+        expect(componentScope.owRefAlias).toBe(componentScope.owRef);
```

Comment:
This only checks that `componentScope.owRefAlias` is `undefined`.
Shouldn't there be similar expectations below ?